media-sound/qmmp/qmmp-2.2.3.ebuild

```diff
@@ -0,0 +1,184 @@
+
+EAPI=8
+
+inherit cmake xdg
+
+DESCRIPTION="Qt6-based audio player with winamp/xmms skins support"
+HOMEPAGE="https://qmmp.ylsoftware.com"
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="
+		https://qmmp.ylsoftware.com/files/${PN}/$(ver_cut 1-2)/${P}.tar.bz2
+		https://downloads.sourceforge.net/${PN}-dev/files/${P}.tar.bz2
+	"
+	KEYWORDS="~amd64 ~x86"
+else
+	inherit subversion
+	local BRANCH_VER="2.2" # NOTE: bump this to $(ver_cut 1-2) for live ebuild
+	ESVN_REPO_URI="svn://svn.code.sf.net/p/qmmp-dev/code/branches/${PN}-${BRANCH_VER}"
+fi
+
+LICENSE="CC-BY-SA-4.0 GPL-2+" # default skin & source code
+SLOT="0"
+# KEYWORDS further up
+IUSE="X aac +alsa analyzer archive bs2b cdda cddb cover crossfade cue curl +dbus
+doc enca ffmpeg flac game gnome jack ladspa libxmp lyrics +mad midi mms mpg123
+mplayer musepack notifier opus oss pipewire projectm pulseaudio qsui qtmedia
+scrobbler shout sid sndfile soxr stereo tray udisks +vorbis wavpack"
+
+REQUIRED_USE="
+	cddb? ( cdda )
+	gnome? ( dbus )
+	notifier? ( X )
+	shout? ( soxr vorbis )
+	udisks? ( dbus )
+"
+
+RDEPEND="
+	dev-qt/qtbase:6[X?,dbus?,gui,network,sqlite,widgets]
+	media-libs/taglib:=
+	X? ( x11-libs/libX11 )
+	aac? ( media-libs/faad2 )
+	alsa? ( media-libs/alsa-lib )
+	archive? ( app-arch/libarchive )
+	bs2b? ( media-libs/libbs2b )
+	cdda? (
+		dev-libs/libcdio:=
+		dev-libs/libcdio-paranoia:=
+	)
+	cddb? ( media-libs/libcddb )
+	curl? ( net-misc/curl )
+	enca? ( app-i18n/enca )
+	ffmpeg? ( media-video/ffmpeg:= )
+	flac? ( media-libs/flac:= )
+	game? ( media-libs/game-music-emu )
+	jack? (	virtual/jack )
+	ladspa? ( media-plugins/cmt-plugins )
+	libxmp? ( media-libs/libxmp )
+	mad? ( media-libs/libmad )
+	midi? ( media-sound/wildmidi )
+	mms? ( media-libs/libmms )
+	mpg123? ( media-sound/mpg123 )
+	mplayer? ( media-video/mplayer )
+	musepack? ( >=media-sound/musepack-tools-444 )
+	opus? ( media-libs/opusfile )
+	pipewire? ( media-video/pipewire:= )
+	projectm? (
+		dev-qt/qtbase:6[-gles2-only,opengl]
+		media-libs/libprojectm:=
+	)
+	pulseaudio? ( media-libs/libpulse )
+	qtmedia? ( dev-qt/qtmultimedia:6 )
+	scrobbler? ( net-misc/curl )
+	shout? ( media-libs/libshout )
+	sid? ( >=media-libs/libsidplayfp-1.1.0:= )
+	sndfile? ( media-libs/libsndfile )
+	soxr? ( media-libs/soxr )
+	udisks? ( sys-fs/udisks:2 )
+	vorbis? (
+		media-libs/libogg
+		media-libs/libvorbis
+	)
+	wavpack? ( media-sound/wavpack )
+"
+DEPEND="
+	${RDEPEND}
+	X? ( x11-base/xorg-proto )
+	doc? ( app-text/doxygen )
+"
+BDEPEND="dev-qt/qttools:6[linguist]"
+
+DOCS=( AUTHORS ChangeLog README )
+
+PATCHES=( "${FILESDIR}"/${P}_build-with-projectm4.patch )
+
+src_configure() {
+	local mycmakeargs=( # listed in readme
+		# potential USE flags
+		-DUSE_LIBRCD=OFF
+		-DUSE_OSS4=OFF
+
+		# explicit defaults
+		-DUSE_COPYPASTE=ON
+		-DUSE_DIR_ASSOC=ON
+		-DUSE_FILEOPS=ON
+		-DUSE_MONOTOSTEREO=ON
+		-DUSE_NULL=ON
+		-DUSE_QMMP_DIALOG=ON
+		-DUSE_TRACKCHANGE=ON
+		-DUSE_TWO_PANEL_DIALOG=ON
```

Comment:
I doubt we need this because these options are depless and there's already so many useflags. Idem for "windows specific stuff".